preview/Win7Msix/Win7MSIXInstaller/InstallUI.hpp

```diff
@@ -22,6 +22,7 @@ static HWND g_CancelbuttonHWnd = NULL;
 static HWND g_LaunchbuttonHWnd = NULL;
 static bool g_launchCheckBoxState = true; /// launch checkbox is checked by default
 static bool g_installed = false;
+static PCWSTR g_installOrUpdateText = L"Install "; /// Default button and install screen UI text is 'Install'
```

Comment:
>g_installOrUpdateText [](start = 14, length = 21)

i'd prefer this to be part of the class rather than global

Reply:
Moved to UI class. Called checking for an update after SetWindowLongPtr() and called ChangeButtonText later to set button text to 'Update' if the operation is an update.